<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Venster</class>
 <widget class="QDialog" name="Venster">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>250</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Minesweeper</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>img/mijnb.ico</normaloff>img/mijnb.ico</iconset>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QFrame" name="labelFrame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>180</width>
     <height>180</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLabel" name="mijnbLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>img/mijnb.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="mOverLabel">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>215</y>
     <width>35</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <stylestrategy>PreferAntialias</stylestrategy>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QLabel" name="tOverLabel">
   <property name="geometry">
    <rect>
     <x>145</x>
     <y>215</y>
     <width>35</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <stylestrategy>PreferAntialias</stylestrategy>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QLabel" name="klokLabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>210</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>img/klok.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="grootteLabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>60</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Size</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="kleinLabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>90</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="text">
    <string>&amp;Small
10 mines
9x9 squares</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="medLabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>110</y>
     <width>101</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="text">
    <string>&amp;Medium
40 mines
16x16 squares</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="grootLabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>170</y>
     <width>101</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="text">
    <string>&amp;Large
99 mines
16x30 squares</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="aangLabel">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>50</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="text">
    <string>&amp;Custom:</string>
   </property>
  </widget>
  <widget class="QLabel" name="hoogteLabel">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>375</x>
     <y>75</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="text">
    <string>Height (9-24):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="hoogteEdit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>75</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="inputMask">
    <string>##</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>2</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="breedteEdit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>100</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="inputMask">
    <string>##</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>2</number>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="breedteLabel">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>375</x>
     <y>100</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="text">
    <string>Width (9-30):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="mijnenEdit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>125</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="inputMask">
    <string>###</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>3</number>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="mijnenLabel">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>375</x>
     <y>125</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="text">
    <string>Mines</string>
   </property>
  </widget>
  <widget class="QPushButton" name="grootteBtn">
   <property name="geometry">
    <rect>
     <x>385</x>
     <y>195</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Start</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>hoogteEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>breedteEdit</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>476</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>475</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>breedteEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>mijnenEdit</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>465</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>465</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hoogteEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>aangLabel</receiver>
   <slot>toggle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>485</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>373</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>breedteEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>aangLabel</receiver>
   <slot>toggle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>478</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>391</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mijnenEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>aangLabel</receiver>
   <slot>toggle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>476</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>396</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hoogteEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>breedteEdit</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>474</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>474</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>breedteEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>mijnenEdit</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>474</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>474</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
